agent/taskresource/ssmsecret/ssmsecretstatus.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package ssmsecret
+
+import (
+	"errors"
+	"strings"
+
+	resourcestatus "github.com/aws/amazon-ecs-agent/agent/taskresource/status"
+)
+
+type SSMSecretStatus resourcestatus.ResourceStatus
+
+const (
+	// is the zero state of a task resource
+	SSMSecretStatusNone SSMSecretStatus = iota
+	// represents a task resource which has been created
+	SSMSecretStatusCreated
```

Comment:
This can just be `SSMSecretCreated` and `SSMSecretRemoved`

Reply:
Why we do not need SSMSecretNone? I need this status when I initialize secret resource.

Reply:
oh, what I meant was we can leave out `Status` in the enum value. yes, we would need `None` as well.